### PR TITLE
Native quarkus

### PR DESCRIPTION
**Pull request by Delawen**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,63 +1,51 @@
...
+FROM node:16 as FRONTEND
 WORKDIR /app
-RUN git clone --depth 1 --branch ${ui_tag} https://github.com/KaotoIO/kaoto-ui.git 
-RUN git clone --depth 1 --branch ${api_tag} https://github.com/KaotoIO/kaoto-backend.git 
-
+RUN git clone --depth 1 https://github.com/KaotoIO/kaoto-ui.git 
 WORKDIR /app/kaoto-ui
-RUN yarn install --mode=skip-build
-
-RUN KAOTO_API="" yarn run build
```
Comment (mmelko) [inline Dockerfile:12]: this cannot be removed. It overrides the default defined in webpack config.

File: Dockerfile [hunk 2]
```diff
@@ -1,63 +1,51 @@
-FROM node:16 as sourcebuild
-ARG ui_tag=main
-ARG api_tag=main
-
+# Frontend build
+FROM node:16 as FRONTEND
 WORKDIR /app
-RUN git clone --depth 1 --branch ${ui_tag} https://github.com/KaotoIO/kaoto-ui.git 
-RUN git clone --depth 1 --branch ${api_tag} https://github.com/KaotoIO/kaoto-backend.git
```
Comment (mmelko) [inline Dockerfile:7]: keep --branch parameters please

File: Dockerfile [hunk 3]
```diff
@@ -1,63 +1,51 @@
...
-    && chown 1001 /deployments \
-    && chmod "g+rwX" /deployments \
-    && chown 1001:root /deployments \
-    && curl https://repo1.maven.org/maven2/io/fabric8/run-java-sh/${RUN_JAVA_VERSION}/run-java-sh-${RUN_JAVA_VERSION}-sh.sh -o /deployments/run-java.sh \
-    && chown 1001 /deployments/run-java.sh \
-    && chmod 540 /deployments/run-java.sh \
-    && echo "securerandom.source=file:/dev/urandom" >> /etc/alternatives/jre/conf/security/java.security
-
-# Configure the JAVA_OPTIONS, you can add -XshowSettings:vm to also display the heap size.
-ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager -Dquarkus.kubernetes-client.trust-certs=true"
```
Comment (mmelko) [inline Dockerfile:53]: how are in native configured these properties ? it can be necessary for the running with k8

File: Dockerfile (hunk 2 above)
Comment (Delawen) [reply Dockerfile:7]: Adding them now!

File: Dockerfile (hunk 1 above)
Comment (Delawen) [reply Dockerfile:12]: The `skip-build` or the `KAOTO_API`?

KAOTO_API is restored on line 7. Maybe I should add skip-build too.

File: Dockerfile (hunk 3 above)
Comment (Delawen) [reply Dockerfile:53]: ```CMD ["./application", "-Dquarkus.http.host=0.0.0.0"]```

**Comment by mmelko** [comment]
LGTM 👍

**Pull request merged**